blog/_posts/2024-11-15-dimensionality-reduction-for-flu-and-sars-cov-2.md

```diff
@@ -0,0 +1,171 @@
+
+<figure>
+    <img src="/images/blog/cartography-original-whiteboard-sketch-for-project.jpg" alt="Original whiteboard sketch of Sravani's summer internship project" />
+    <figcaption><strong>Figure 3. The original whiteboard sketch of Sravani's summer internship project showing the family tree of a single type of virus (top left) and sketches of what maps from different methods might look like including PCA, t-SNE, and UMAP.</strong> We wanted this figure to be interactive, so viewers could select viruses in one panel to highlight their corresponding positions in other panels.</figcaption>
+</figure>
+
+To make the project more interesting from a data science perspective, we agreed that the visualization should be interactive, so we could select viruses in the tree or one of the maps and the same viruses would get highlighted in the other panels of the figure.
+
+Over 2 months, Sravani learned how to:
+
+  - work with virus mutation data
+  - build virus trees from mutation data
+  - calculate distances between pairs of virus copies based on their mutations
+  - make two-dimensional maps from mutation data using methods with exciting names like principal components analysis (PCA), multidimensional scaling (MDS), t-distributed stochastic neighbor embedding (t-SNE), and uniform manifold approximation and projection (UMAP)
+  - plot trees and maps in a single interactive figure that allowed us to highlight bits of the tree or a map and see the same viruses in the other parts of the figure
+
+By August 2019, Sravani had made the prototype below (Figure 4) from mutations in a type of seasonal flu called “H3N2” which causes the most hospitalizations and deaths each year.
+
+<figure>
+    <img src="/images/blog/cartography-final-internship-prototype.png" alt="Sravani's final internship prototype showing maps based on flu mutations" />
+    <figcaption><strong>Figure 4. Static view of Sravani's final internship prototype showing individual viruses in a family tree (top left) and corresponding positions of the same viruses in maps based on flu mutations including PCA (top middle), MDS (top right), t-SNE (bottom left), and UMAP (bottom right).</strong> Viruses in two specific groups from the tree (blue and orange) have been selected to show how their placement in the tree compares to their placement in the maps.</figcaption>
+</figure>
+
+The prototype revealed some interesting patterns:
+
+  1. Most of the maps placed pairs of viruses with the same mutations closer together than pairs with different mutations.
+  2. Some of the maps (like MDS's) actually acted like a real map with the distance between viruses on the map matching exactly the number of mutations that differed between those viruses.
```

Comment:
```suggestion
  2. Some of the mapping methods (like MDS) actually acted like a real map with the distance between viruses on the map matching exactly the number of mutations that differed between those viruses.
```

Reply:
I'm going to skip this suggestion only because it's the maps that act like real maps and not the methods that act like maps.

Reply:
Totally fair — this is the point where I confess that my real motivation here was trying to find a phrasing that allowed the removal of the greengrocer's apostrophe's (sic) in the parenthetical map names... but maybe if you're considering that map a possession of the mapping technique, this isn't actually incorrect 🤔

Reply:
That makes sense. I'm ok with saying that a map belongs to a method as in "MDS's map".